src/gl/implementation/gl-objects-textures-with_2d_loader.adb

```diff
@@ -47,6 +47,20 @@ package body GL.Objects.Textures.With_2D_Loader is
       Raise_Exception_On_OpenGL_Error;
    end Load_From_Data;
 
+   procedure Load_SubImage_From_Data
```

Comment:
I'd like to have this changed to `Load_Sub_Image_From_Data` so it's consistent with all other names.

Reply:
Fine.
Should I implement corresponding Load_Sub_Image_From_Data for glTexSubImage1D and glTexSubImage3D?

Reply:
Would certainly be good to have them, please do.

Reply:
OK, tomorrow's job.